pkgs/development/ocaml-modules/gitlab/unix.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildDunePackage, gitlab
+, cmdliner, cohttp, cohttp-lwt-unix
+, tls, lwt, stringext, alcotest
+}:
+
+buildDunePackage {
+  pname = "gitlab-unix";
+  inherit (gitlab) version src;
+
+  duneVersion = "3";
+  minimalOCamlVersion = "4.08";
+
+  postPatch = ''
+    substituteInPlace unix/dune --replace 'gitlab bytes' 'gitlab'
+  '';
+
+  propagatedBuildInputs = [
+    gitlab
+    cmdliner
+    cohttp
+    cohttp-lwt-unix
+    tls
+    lwt
+    stringext
+  ];
```

Comment:
`requires = "cmdliner cohttp-lwt-unix gitlab lwt.unix"`

Rest should go into `buildInputs`

Reply:
This is a great tip. I will make the changes now, and run `nixfmt-rfc-style`.